jjb/sesdev.yaml

```diff
@@ -0,0 +1,29 @@
+- project:
+    name: sesdev
+    repo-name: sesdev
+    repo-owner: SUSE
+    repo-credentials: susebot
+    jobs:
+      - '{name}-integration'
```

Comment:
Not sure 'project' section is required, since it is just a single job, though, do you use as a placeholder for 'variables/contants'?

Reply:
Secret plan was to add more jobs (eg. testing with other parameters like not only octopus, but also nautilus. Or even better, also with CaaSP). So we would have multiple jobs here. Does that make sense?

Reply:
yeh